src/services/dynamodb.js

```diff
@@ -0,0 +1,13 @@
+const AWS = require("aws-sdk");
+
+const dynamoDB = new AWS.DynamoDB.DocumentClient({
+  accessKeyId: process.env.S3_ACCESS_KEY,
+  secretAccessKey: process.env.S3_SECRET_ACCESS_KEY,
+  region: process.env.AWS_REGION
```

Comment:
this can be hard-codded to `eu-west-2`
```suggestion
  region: 'eu-west-2'
```